src/main/java/binnie/core/genetics/BreedingSystem.java

```diff
@@ -145,12 +144,15 @@ public void calculateArrays() {
 				allBranches.add(branch);
 			}
 		}
+
 		if (getSpeciesRoot().getMutations(false) != null) {
 			Set<IMutation> mutations = new LinkedHashSet<>();
 			mutations.addAll(getSpeciesRoot().getMutations(false));
 			if (this == Binnie.Genetics.beeBreedingSystem) {
-				mutations.addAll(ExtraBeeMutation.mutations);
+				// TODO !!! check for needed !!!
```

Comment:
The lines were changed/introduced in a commit called "Fixes and stuff for Pre5" (d286440 2015-01-20), but the commit was huge (77 changed files), so I don't understand the reasons for and the effects of this change.